docs/changelogs/v0.21.md

```diff
@@ -56,6 +56,18 @@ backend for other software (like
 etc) without the usual risks associated with hosting deserialized data behind
 third-party CIDs.
 
+#### `client/rpc` migration of `go-ipfs-http-client`
+
+The [`go-ipfs-http-client`](https://github.com/ipfs/go-ipfs-http-client) RPC has
+been migrated into [`client/rpc`](./client/rpc).
+
+With this change the two will be kept in sync, in some previous releases we
+updated the CoreAPI with new Kubo features but forgot to port thoses to the
+http-client, making it impossible to use them together with the same coreapi
+version.
+
+**TODO(@Jorropo)**: add link to `boxo-migrate` once support for rewriting this import path has been added
```

Comment:
FYSA: I merged this PR assuming this will be updated before the release with the correct instructions. Feel free to ping me to review that.